src/Spain/Mews.Fiscalizations.Spain/Model/Response/ErrorResult.cs

```diff
@@ -8,12 +9,20 @@ private ErrorResult(string code, string message)
         {
             Code = code;
             Message = message;
+            ErrorType = ConvertErrorType(message);
         }
 
         public string Code { get; }
 
+        public ErrorType ErrorType { get; }
         public string Message { get; }
 
         public static ErrorResult Create(string code, string message) => new ErrorResult(code, message);
+
+        private ErrorType ConvertErrorType(string message)
+        {
+            return message.Match(
+                "Codigo[401].Certificado revocado", _ => ErrorType.CertificateRevoked);
```

Comment:
```suggestion
                "Codigo[401].Certificado revocado", _ => ErrorType.CertificateRevoked,
                _ => ErrorType.Unknown
       )
```

Reply:
I'm sure at some point we will extend this, but for now let's make sure we return Unknown error when we're not sure what it is.